### PR TITLE
fleetctl 0.11.8

### PR DESCRIPTION
**Pull request by jasperla**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

Tested against CoreOS running fleetd 0.11.8.

### CODE REVIEW

File: Formula/fleetctl.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Fleetctl < Formula
   desc "Distributed init system"
   homepage "https://github.com/coreos/fleet"
-  url "https://github.com/coreos/fleet/archive/v0.11.7.tar.gz"
-  sha256 "5c838059826d6cde1183554701d3af93619980c299fe9d1365588a30a4ca6cc8"
+  url "https://github.com/coreos/fleet/archive/v0.11.8.tar.gz"
+  sha256 "22f2f40c1c2938504b31e9dbb54eb6eb54569458a4dffe5e0020e917a5e1f66f"
```
Comment (zmwangx) [inline Formula/fleetctl.rb]: Blank line unnecessary here.

**Comment by zmwangx** [comment]
I'd rather have a test like this:

```ruby
  test do
    assert_match version.to_s, shell_output("#{bin}/fleetctl -version")
  end
```

**Comment by jasperla** [comment]
@zmwangx thanks, I've updated it accordingly.

**Comment by zmwangx** [comment]
Cool, I think this is mergeable once CI is green (and it should be green). LLVM is currently building though which will take about two hours, so please wait patiently.

**Comment by zmwangx** [comment]
Merged as d483dbb. Thank you for your contribution to Homebrew!

**Comment by jasperla** [comment]
Cheers, thanks for the merge!

**Pull request closed without merging**